src/collect-projects.rst

```diff
@@ -27,3 +33,24 @@ Adding a new Project
 #. Tap the :ref:`Project <collect-projects>` icon on the upper right of the :ref:`Main Menu <main-menu>` screen
 #. Tap :guilabel:`Add project`
 #. Scan a QR code or tap :guilabel:`Manually add project details` to set the server for the new project
+
+.. warning::
+  If a Project is added with the same URL as an existing one, Collect will warn the user before creation is finalized. This is to prevent users from accidentally adding the same Project more than once.
```

Comment:
"Collect will warn the user before proceeding" is clearer to me.

"adding the same Project multiple times" is clearer to me too, but it's more a nit pick.